.stylelintrc

```diff
@@ -3,7 +3,8 @@
     "stylelint-config-standard",
   ],
   "rules": {
-    "selector-type-no-unknown": null
+    "selector-type-no-unknown": null,
+    "at-rule-no-unknown": null
```

Comment:
I'd be consistent and also add it here. You may also want to consider sorting the list alphabetically. Those are just some practices I started to use because it's easier to deal with when things get longer and you're searching for something.

Reply:
@ekohl Added another comma and also sorted alphabetically. Thanks for the review! :)